scania-digital-design-system/sdds
# fix(toggle): refactor event names

**Describe pull-request**  
Refactored event names, removed aria props and unused label prop.

**Solving issue**  
Fixes: [AB#3469](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/3469)

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Toggle -> Web Component
3. Check the readme for the event names.

**Suggested test steps**
